source/administration/announcement-banner.rst

```diff
@@ -0,0 +1,22 @@
+Announcement Banner
+===================
+
+Available in `Enterprise Edition E10 and higher <https://about.mattermost.com/pricing/>`_.
+
+In Mattermost Enterprise Edition, an announcement banner is available that allows System Admins to display a notice that's visible to all users on the system.
+
+.. image:: ../images/announcement-banner-1106x272.png
+  :width: 1106
+  :height: 272
+  :alt: Shows the announcement banner at the top of the user's screen.
+
+By default, users can dismiss the banner until they log in again or until you update the banner. You can update the banner by either changing the text of the banner or by re-enabling the banner after it has been disabled. You can also control the text color and the background color. A setting in the System Console allows you to prevent users from dismissing the banner.
```

Comment:
Would it be possible to add something here that's more around use cases than just describing functionality? 

I think we were trying to move towards phrasing docs in a way that answer user questions as opposed to just listing features, and including information about why people would use the feature might help with that.